src/App.jsx

```diff
@@ -20,12 +20,18 @@ function App() {
     setTodoList([...todoList, newTodo])
   }
 
+  const removeTodo = (id) => {
+    const newToDoList = todoList.filter(
```

Comment:
Since this variable is only used in one place, another option could be to invoke the filter function directly in the arguments for setTodoList. This is definitely optional – and using a variable to store the results of the filter function might be slightly more readable – but it's worth considering as an alternative.